addon/components/radio-button.js

```diff
@@ -1,19 +1,25 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
+import { equal } from '@ember/object/computed';
 
 export default Component.extend({
   tagName: 'label',
   classNames: ['btn', 'upf-radio-btn'],
-  classNameBindings: ['isChecked:active'],
+  classNameBindings: ['isChecked:active', 'isDisabled:disabled'],
```

Comment:
you can just do `disabled:disabled` and no need to define a `isDisabled` computed.